scripts/src/report/report_info.py

```diff
@@ -0,0 +1,86 @@
+
+import os
+import sys
+import docker
+import logging
+import json
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+REPORT_ANNOTATIONS = "annotations"
+REPORT_RESULTS = "results"
+REPORT_DIGESTS = "digests"
+REPORT_METADATA= "metadata"
+
+def getReportInfo(report_path,info_type,profile_type,profile_version):
+
+    docker_command = "report " + info_type + " charts/"+os.path.basename(report_path)
+
+    set_values = ""
+    if profile_type:
+        set_values = "profile.vendortype="+profile_type
+    if profile_version:
+        if set_values:
+            set_values = set_values + ","
+        set_values = set_values + "profile.version="+profile_version
+
+    if set_values:
+        docker_command += " --set " + set_values
+
+    client = docker.from_env()
+    report_directory = os.path.dirname(os.path.abspath(report_path))
+    output = client.containers.run(os.environ.get("VERIFIER_IMAGE"),docker_command,stdin_open=True,tty=True,stderr=True,volumes={report_directory: {'bind': '/charts/', 'mode': 'rw'}})
+    report_out = json.loads(output)
+
+    if not info_type in report_out:
+        print(f"Error extracting {info_type} from the report:", r.strip())
+        sys.exit(1)
+
+    if info_type == REPORT_ANNOTATIONS:
+        annotations = {}
+        for report_annotation in report_out[REPORT_ANNOTATIONS]:
+            annotations[report_annotation["name"]] = report_annotation["value"]
+
+        return annotations
+
+    return report_out[info_type]
+
+
+def getReportAnnotations(report_path):
+    annotations = getReportInfo(report_path,REPORT_ANNOTATIONS,"","")
```

Comment:
might be better to call `chart-verifier report all` with the corresponding profile once and parse the result, instead of calling the docker command from each of the functions.

Reply:
Would then need ti store the results to serve different requests, not sure how to do that in python.

Reply:
I was thinking about setting a private variable in `report_info.py` and checks if this variable is empty each `getReportInfo` run and skip docker run if the info is already retrieved. But then I realized some runs contain vendor type and version, it might require setting vendor type & version in the first run. I don't disagree with the current approach because i don't think the overhead will be large.

Reply:
OK, lets leave as-is for now, and consider changing in the future.